frontend/src/components/EntityBulkAdd.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div v-for="num in entityFormComponentsCountRange" :key="`ef-${num}`">
+    <EntityForm :ref="`entity-form-${num}`" :schema="schema"
+                :attributes="attributes" :batch-mode="true" @save-all="saveAll"/>
+  </div>
+  <div class="container mt-2">
+    <button class="btn btn-outline-secondary w-100" @click="addEntityForm">
+      <i class='eos-icons'>add_circle</i>
+      Add more
```

Comment:
I noticed that clicking add more always triggers an API request, seems unnecessary since it's all just a copy of the same entity.

Reply:
yeah, I believe that's because the nested components do requests.. here additional refactoring should be made also

Reply:
yes, it's no cause by your changes directly, tho I still felt like mentioning it since it directly impacts the new feature

Reply:
Yeah, fair point.. I would like to combine it with future refactoring of entity form in another branch and issue (if you don't mind). I will leave myself a note in order to not forget to create the task